stylesheets/_component.navigation.scss

```diff
@@ -219,6 +219,7 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
 .nav-primary {
     background-color: $nav-color-light;
     box-shadow: inset 0 0 0 $nav-color-dark;
+    overflow-y: hidden;
```

Comment:
It will prevent scrolling of primary nav in general. What is the viewpoint you see, is the height of primary nav you get.